DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/Centre/Administrator/AdministratorControllerTests.cs

```diff
@@ -188,5 +188,36 @@ public void UnlockAccount_unlocks_account_and_returns_to_page()
             A.CallTo(() => userDataService.UpdateAdminUserFailedLoginCount(1, 0)).MustHaveHappened();
             result.Should().BeRedirectToActionResult().WithActionName("Index");
         }
+
+        [Test]
+        public void DeactivateAdminUser_does_not_deactivate_admin_user_without_confirmation()
+        {
+            // Given
+            var adminUser = new AdminUser() { Id = 1 };
+            var removeViewModel = new DeactivateAdminViewModel(adminUser) { Confirm = false };
+            const string expectedErrorMessage = "You must confirm before deactivating this account.";
+
+            // When
+            var result = administratorController.DeactivateAdmin(1, removeViewModel);
+
+            // Then
+            result.Should().BeViewResult().WithDefaultViewName().ModelAs<DeactivateAdminViewModel>();

```

Comment:
It would be good to have an additional test condition to make sure a call to the data service's deactivate method is not made. 

As this Then section is getting larger with multiple different assertions, we should probably wrap it in an AssertionScope.

Reply:
Added another check as suggested

Reply:
Which check are you referring to? I was expecting to see something like:
`            A.CallTo(() => userDataService.DeactivateAdmin(A<int>._)).MustNotHaveHappened();`